build/docker-image/self-signer-cert-utility/Dockerfile

```diff
@@ -18,13 +18,16 @@ COPY cmd/ cmd/
 COPY pkg/ pkg/
 
 # Build the binary self-signer utility
-RUN go build -a -installsuffix cgo -o self-signer cmd/main.go
+RUN go build -o self-signer cmd/main.go
 
 # Install the cockroach CLI
```

Comment:
A nit. "cockroach CLI" means `cockroach-cli`, which is a separate binary and doesn't contain any "server" code. Maybe adjust the comment to something like "Install the cockroach binary"?